tests/auth_tests/test_hashers.py

```diff
@@ -596,3 +596,22 @@ def setter(password):
             self.assertTrue(state['upgraded'])
         finally:
             setattr(hasher, attr, old_value)
+
+
+@override_settings(PASSWORD_HASHERS=PASSWORD_HASHERS)
+class TestUtilsHashPassScrypt(SimpleTestCase):
+
+    def test_scrypt(self):
+        encoded = make_password('lètmein', 'seasalt', 'scrypt')
+        self.assertEqual(
```

Comment:
You can use something like this (wrapping lines at 79 chars):
```
self.assertEqual(
    encoded, 
    'scrypt$16384$seasalt$Qj3+9PPyRjSJIebHnG81TMjsqtaIGxNQG/aEB/NYaf'
    'TJ7tibgfYz71m0ldQESkXFRkdVCBhhY8mx7rQwite/Pw==$8$1'
)
```

Reply:
I adjusted the string to match that in the most recent commit.